Makefile

```diff
@@ -49,6 +49,10 @@ db: python
 dev: build/manifest.json python
 	@tox -qe dev
 
+.PHONY: devssl
+devssl: build/manifest.json python
+	@env H_GUNICORN_CERTFILE=.tlscert.pem H_GUNICORN_KEYFILE=.tlskey.pem APP_URL='https://localhost:5000' WEBSOCKET_URL='wss://localhost:5001/ws' tox -qe dev
```

Comment:
`make devssl` replaces [running `hypothesis devserver --https`](https://h.readthedocs.io/en/latest/developing/ssl/) (and the docs are updated)